src/demo/models/guide.js

```diff
@@ -303,12 +303,12 @@ const guides = [
   },
   {
     id: 'fishshort-words-training-pause1',
-    text: `Nice work.  Keep training.`,
+    text: `Do you think A.I. has enough training data?  You could click Continue to find out.`,
```

Comment:
We normally use "can" instead of "could", but perhaps this way it feels less like a suggestion, and more like an idea?

Reply:
fixed, good catch!